src/schema/v2/requestLocation.ts

```diff
@@ -20,6 +23,8 @@ const logRateHeaders = (headers) => {
 export const RequestLocationType = new GraphQLObjectType<any, ResolverContext>({
   name: "RequestLocation",
   fields: () => ({
+    cached,
+    id: { type: new GraphQLNonNull(GraphQLID) },
```

Comment:
@dzucconi - i just ran into this when using the fragment spread technique for updating a view post mutation and would have had no clue how to achieve had this PR not been opened the other day... How did you know to do this? Like are there some docs we could point at somewhere?

Reply:
I guess there's this: https://relay.dev/docs/principles-and-architecture/thinking-in-graphql/#caching-a-graph

I actually thought that Relay was using `${_typename}:${id}` like Apollo but instead it relies on globally unique IDs. Which means we should actually use this instead of just base64 encoding the IP address: https://github.com/artsy/metaphysics/blob/0e7202ed5899e9dee465ddded88950138ee7e5f3/src/schema/v2/object_identification.ts#L166-L178

Reply:
nice, ty 👍